contracts/proposal/SoftwareUpgradeProposal.sol

```diff
@@ -3,31 +3,39 @@ pragma solidity 0.8.27;
 
 import {Cancelable} from "./base/Cancelable.sol";
 import {DelegatecallExecutableProposal} from "./base/DelegatecallExecutableProposal.sol";
-
-/// @notice An interface to update this contract to a destination address
-interface Upgradability {
-    function upgradeTo(address newImplementation) external;
-}
+import {ITransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
```

Comment:
Since we already discussed we do this because of our SFC/etc, use https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/master/contracts/proxy/utils/UUPSUpgradeable.sol instead of TransparentUpgradeableProxy.